torch/csrc/jit/ir_views.h

```diff
@@ -80,9 +86,33 @@ struct LoopView {
     return node_;
   }
 
+  // adjust index_ordering by adding indices 0 - thorugh adjust, and
+  // incrementing all existing inputs by adjust
+  std::vector<size_t> adjustIndices(
```

Comment:
nit: this is not a method of LoopView because it doesn't use any members of loop view.